### PR TITLE
reimplemented survey using the new plugin

### PR DESCRIPTION
**Pull request by bbonf**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,9 +28,10 @@
         <link href=https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/0.1/fonts.css rel="stylesheet" type="text/css"/>
 
         <!-- Uil OTS libraries -->
-        <script type="module" src="https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/1.0/jspsych-uil-utils-import.js"></script>
+        <script type="module" src="https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/1.1/jspsych-uil-utils-import.js"></script>
 
          <!-- Uil OTS organised  -->
+        <script src="https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/1.1/jspsych-plugins/plugin-survey.js"></script>
```
Comment (maartenuni) [inline index.html:32]: When we use our own survey plugin we can probably remove the plugins from jspsych on lines 19 - 22 of this file

File: survey.js [hunk 2]
```diff
@@ -60,201 +60,128 @@ class ParticipantInfo {
...
+	return false
+    },
+    html: `
+    <div style="text-align: left">
+        <p>Were you born and raised in a multilingual environment?</p>
+        <label><input type="radio" name="multilingual" value="yes" required/>Yes</label>
+        <label><input type="radio" name="multilingual" value="no" required/>No</label>
+
+	<p>Which hand do you prefer to write with?
+            <span class="info-toggle">🛈</span>
```
Comment (maartenuni) [inline survey.js]: I see encoding/decoding of text/symbols errors here

File: survey.js [hunk 3]
```diff
@@ -60,201 +60,128 @@ class ParticipantInfo {
...
+                certain response with your dominant hand.
+            </span>
+        </p>
+	<div>
+	    <label><input type="radio" name="handedness" value="right" required/>Right</label>
+	    <label><input type="radio" name="handedness" value="left" required/>Left</label>
+	</div>
+
+        <p>What is your biological sex?
+            <span class="info-toggle">🛈</span>
```
Comment (maartenuni) [inline survey.js]: I see encoding/decoding of text/symbols errors here

**Review by maartenuni — COMMENTED** [review]
I've added a few comments, we've discussed these. Further everything looks nice!

File: index.html (hunk 1 above)
Comment (bbonf) [reply index.html:32]: jspsych-survey-text and jspsych-survey-multi-select are still used for the feedback and consent screens

File: survey.js (hunk 2 above)
Comment (bbonf) [reply survey.js]: If you want to test the updated version, you have to manually include the dev version of utils (https://web-experiments.lab.hum.uu.nl/jspsych/uil-utils/dev/jspsych-plugins/plugin-survey.js). I'll make a new release later

**Review by maartenuni — APPROVED** [review]
It looks good! Is it worth the trouble to replace the jspsych-surveys from the with our own survey, or do they serve different purposes?

**Comment by bbonf** [comment]
I don't think there's much to gain from it. The survey plugin was intended to make it easier to edit the survey steps, but the consent and feedback steps shouldn't really be edited much.

**Pull request merged**